poulpe-service/src/test/java/org/jtalks/poulpe/service/transactional/TransactionalUserServiceTest.java

```diff
@@ -264,42 +264,42 @@ public void testUpdateUsers(){
     
     @Test
     public void authenticate() throws NotFoundException {
-        when(userDao.findPoulpeUsersBySearchRequest(any(UserSearchRequest.class)))
-                .thenReturn(Collections.singletonList(POULPE_USER));
+        when(userDao.getByUsernameAndPasswordHash(anyString(), anyString()))
+                .thenReturn(Arrays.asList(POULPE_USER, ANOTHER_USER));
 
         assertEquals(userService.authenticate(USERNAME, HASED_PASSWORD), POULPE_USER);
 
-        verify(userDao).findPoulpeUsersBySearchRequest(any(UserSearchRequest.class));
+        verify(userDao).getByUsernameAndPasswordHash(anyString(), anyString());
         verify(userDao, never()).getByUsername(eq(USERNAME));
     }
 
     @Test
     public void authenticateCaseSensitise() throws NotFoundException {
-        when(userDao.findPoulpeUsersBySearchRequest(any(UserSearchRequest.class)))
+        when(userDao.getByUsernameAndPasswordHash(anyString(), anyString()))
                 .thenReturn(Arrays.asList(POULPE_USER, ANOTHER_USER));
 
         assertEquals(userService.authenticate(USERNAME, HASED_PASSWORD), POULPE_USER);
 
-        verify(userDao).findPoulpeUsersBySearchRequest(any(UserSearchRequest.class));
+        verify(userDao).getByUsernameAndPasswordHash(anyString(), anyString());
     }
 
     @Test
     public void authenticateCaseSensitise_whenUsernameNotFound() throws NotFoundException {
         String searchUsername = StringUtils.capitalize(USERNAME);
-        when(userDao.findPoulpeUsersBySearchRequest(any(UserSearchRequest.class)))
+        when(userDao.getByUsernameAndPasswordHash(anyString(), anyString()))
                 .thenReturn(Arrays.asList(POULPE_USER, ANOTHER_USER));
 
         try {
             userService.authenticate(searchUsername, HASED_PASSWORD);
             fail("NotFoundException should be thrown when case sensitive username is not found");
         } catch (NotFoundException e) {
         }
-        verify(userDao).findPoulpeUsersBySearchRequest(any(UserSearchRequest.class));
+        verify(userDao).getByUsernameAndPasswordHash(anyString(), anyString());
     }
 
     @Test
         public void authenticate_whenUsernameNotFound() throws NotFoundException {
-        when(userDao.findPoulpeUsersBySearchRequest(any(UserSearchRequest.class)))
+        when(userDao.getByUsernameAndPasswordHash(anyString(), anyString()))
                 .thenReturn(Collections.<PoulpeUser>emptyList());
 
         try {
```

Comment:
Instead of try/catch JUnit has:
- expected = ExceptionClass.class
- Or there are special `@Rule`'s